cli/src/commands/config/edit.rs

```diff
@@ -40,6 +42,34 @@ pub fn cmd_config_edit(
     if !file.path().exists() {
         file.save()?;
     }
-    editor.edit_file(file.path())?;
+
+    // Editing again and again until
+    // 1. The config is OK
+    // 2. The user restore previous one
+    loop {
+        editor.edit_file(file.path())?;
+
+        // Trying to load back config. If error, prompt to continue editing
+        if let Err(e) = ConfigLayer::load_from_file(file.layer().source, file.path().to_path_buf())
+        {
+            writeln!(
```

Comment:
nit: this could be better expressed as `let Err(e) = ... else { break; };`, but I don't feel strongly.

Reply:
I left the previous explicit if statement as I found it more readable

Reply:
If you like explicit control flow, `match` would look nicer.